### PR TITLE
cleanup broken links

### PR DESCRIPTION
**Pull request by BigEd**

### CODE REVIEW

**Comment by BigEd** [comment]
aiming to fix errors as seen in pull request 189

strongly recommend and prefer that we fix broken links by finding good replacements, using the Internet Archive and perhaps following redirects found there.

File: README.md [hunk 1]
```diff
@@ -242,7 +242,7 @@
 
 [Instapaper](https://web.archive.org/web/20211124170124/https://medium.com/making-instapaper/instapaper-outage-cause-recovery-3c32a7e9cc5f). Also [this](https://blog.instapaper.com/post/157027537441). Limits were hit for a hosted database. It took many hours to migrate over to a new database.
 
-[Intel](https://42gems.com/blog/?p=735). A scripting bug caused the generation of the divider logic in the Pentium to very occasionally produce incorrect results. The bug wasn't caught in testing because of an incorrect assumption in a proof of correctness.
+[Intel](https://42gems.com/blog/?p=735). A scripting bug caused the generation of the divider logic in the Pentium to very occasionally produce incorrect results. The bug wasn't caught in testing because of an incorrect assumption in a proof of correctness. (FDIV bug, 1994.)
```
Comment (icco) [inline README.md:245]: What's this comment?

File: README.md [hunk 2]
```diff
@@ -433,7 +433,7 @@
 * Peter Demin
 * Raul Ochoa
 * Ruairi Carroll
-* [Rui Chen](https://github.com/chenrui333)
+* Rui Chen
```
Comment (icco) [inline README.md:436]: Fixed in #181

**Review by icco — COMMENTED** [review]
One small question, then will merge.

File: README.md (hunk 1 above)
Comment (BigEd) [reply README.md:245]: Oh, yes, that's not a link-fix, sorry.  Added the comment for clarity, as the FDIV bug is a famous case and the entry ought to not that. Likewise the date - this is not a recent Intel problem.  We could even link to Wikipedia. https://en.wikipedia.org/wiki/Pentium_FDIV_bug

Comment (icco) [reply README.md:245]: The parenthetical doesn't add much for me, but if you added something like `See [the Wikipedia article on 1994 FDIV bug](https://en.wikipedia.org/wiki/Pentium_FDIV_bug) for more information.`

Comment (BigEd) [reply README.md]: Good thinking, I've made that edit (hopefully in a way which makes it part of this pull request)

**Review by icco — APPROVED** [review]

**Pull request merged**